ci/perf-test.sh

```diff
@@ -86,6 +86,10 @@ function main {
 }
 
 function anyMatchingTargets {
+    if ! [[ -z "${PERF_BRANCH_OVERRIDE}" ]]; then
+        return 0
+    fi
+
```

Comment:
have you tested this works when you don't set `PERF_BRANCH_OVERRIDE`?

Reply:
I have now